server/fileupdater.mjs

```diff
@@ -0,0 +1,28 @@
+const VERSION = 2;
+
+export default function FileUpdater(state) {
+  const v = state._meta.version;
+  if(v == VERSION)
+    return state;
+
+  for(const id in state) {
+    const w = state[id];
+
+    if(v == 1 && state[id].type == 'button') {
+      let isFirstSelect = true;
+      for(const operation of w.clickRoutine) {
```

Comment:
Would there be a problem if .clickRoutine is undefined?

Reply:
Yes. I'll fix that.